### PR TITLE
Feedback html issues - DE44660

### PR DESCRIPTION
**Pull request by imcdonald-d2l**
### Issues

- Feedback in student view was not displaying HTML properly. This is because we were not saving HTML properly when the new HTML editor was enabled. 
- HTML feedback formatting would be cleared as soon as you blurred the field. This is because we were not updating the field value with the HTML value. This issue was not mentioned in the defect but was found while fixing the above issue.

[Rally Ticket
](https://rally1.rallydev.com/#/57732444928d/custom/373260458992?detail=%2Fdefect%2F605852339239)

### Before
feedback-format-issue

### After
feedback-format-issue-fixed

### CODE REVIEW

File: d2l-rubric-feedback.js [hunk 1]
```diff
@@ -510,14 +510,12 @@ Polymer({
...
 
-	updateAssessmentFeedbackText: function(criterionAssessment) {
+	updateAssessmentFeedbackValue: function(criterionAssessment) {
 		if (!this._feedbackModified && !this._pendingFeedbackSaves) {
 			this.toggleBubble('_feedbackInvalid', false, 'feedback-bubble');
-			if (this.CriterionAssessmentHelper.getIsFeedbackOverridden(criterionAssessment)) {
-				this._feedback = this.CriterionAssessmentHelper.getFeedbackText(criterionAssessment);
-			} else {
-				this._feedback = this.CriterionAssessmentHelper.getFeedbackHtml(criterionAssessment);
-			}
```
Comment (imcdonald-d2l) [inline d2l-rubric-feedback.js:520]: `if` condition is unnecessary since they are returning the exact same thing.

**Review by devpow112 — APPROVED** [review]

File: d2l-rubric-feedback.js [hunk 2]
```diff
@@ -510,14 +510,12 @@ Polymer({
...
+			this._feedback = this.useNewBrightspaceEditor ?
+				this.CriterionAssessmentHelper.getFeedbackHtml(criterionAssessment):
+				this.CriterionAssessmentHelper.getFeedbackText(criterionAssessment);
```
Comment (imcdonald-d2l) [inline d2l-rubric-feedback.js:518]: This fixes the format changes being removed on blur. See gif below showing issue.
format-cleared-on-blur

**Comment by devpow112** [comment]
Minor lint issue. Probably just need to run `npm run format`.
```
/home/runner/work/d2l-rubric/d2l-rubric/d2l-rubric-feedback.js
Error:   517:72  error  Operator ':' must be spaced  space-infix-ops
```

File: d2l-rubric-feedback.js [hunk 3]
```diff
@@ -477,10 +477,10 @@ Polymer({
 			bubbles: true,
 			composed: true,
 		}));
-		return this.CriterionAssessmentHelper.updateFeedbackAsync(() => this.criterionAssessment, value, false).finally(function() {
+		return this.CriterionAssessmentHelper.updateFeedbackAsync(() => this.criterionAssessment, value, this.useNewBrightspaceEditor).finally(function() {
```
Comment (imcdonald-d2l) [inline d2l-rubric-feedback.js:480]: `isHtml` parameter was being hard-coded to false. This parameter needs to update based on `this.useNewBrightspaceEditor`.

See function definition [here](https://github.com/Brightspace/d2l-rubric/blob/master/assessment-helpers/criterion-assessment.js#L84).

**Review by devpow112 — APPROVED** [review]

**Pull request merged**